CHANGELOG.md

```diff
@@ -27,6 +27,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased][]
 
+### Fixed
+
+- Fixed crash in certain circumstances when a key was applied after a
+  groupby (#1639).
+
+
+### Notes
+
+- Thanks to everyone who helped make `datatable` more stable by discovering
+  and reporting bugs that were fixed in this release:
+
+  [antorsae][] (#1639),
+
+
 
 ## [0.8.0][] — 2019-01-04
```

Comment:
Is there any way to change the date for the last release? :)

Reply:
Sure, I'll try to remember to change it in the next PR